modules/configuring-quota-management-for-distributed-workloads.adoc

```diff
@@ -0,0 +1,133 @@
+endif::[]
+* You have sufficient resources. In addition to the base {productname-short} resources, you need 1.1 vCPU and 1.6 GB memory to deploy the distributed workloads infrastructure.
+* The resources are physically available in the cluster.
++
+[NOTE]
+====
+{productname-short} currently supports only a single cluster queue per cluster (that is, homogenous clusters), and only empty resource flavors.
+For more information about Kueue resources, see the link:https://kueue.sigs.k8s.io/docs/concepts/[Kueue documentation].
+====
+
+
+.Procedure
+
+. In a terminal window, if you are not already logged in to your OpenShift cluster as a cluster administrator, log in to the OpenShift CLI as shown in the following example:
++
+[source,subs="+quotes"]
+----
+$ oc login __<openshift_cluster_url>__ -u __<admin_username>__ -p __<password>__
+----
+
+. Create an empty Kueue resource flavor, as follows:
+.. Create a file called `default_flavor.yaml` and populate it with the following content:
++
+.Empty Kueue resource flavor
+[source,bash]
+----
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ResourceFlavor
+metadata:
+  name: default-flavor
+----
+.. Apply the configuration to create the `default-flavor` object:
++
+[source,bash]
+----
+$ oc apply -f default_flavor.yaml
+----
+
+. Create a cluster queue to manage the empty Kueue resource flavor, as follows:
+.. Create a file called `cluster_queue.yaml` and populate it with the following content:
++
+.Example cluster queue
+[source,bash]
+----
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ClusterQueue
+metadata:
+  name: "cluster-queue"
+spec:
+  namespaceSelector: {}  # match all.
+  resourceGroups:
+  - coveredResources: ["cpu", "memory", "nvidia.com/gpu"]
+	flavors:
+	- name: "default-flavor"
+  	  resources:
+   	  - name: "cpu"
+    	nominalQuota: 9
+  	  - name: "memory"
+    	nominalQuota: 36Gi
+  	  - name: "nvidia.com/gpu"
+    	nominalQuota: 5
+----
++
+.. Replace the example quota values (9 CPUs, 36 GiB memory, and 5 NVIDIA GPUs) with the appropriate values for your cluster queue.
+The cluster queue will start a distributed workload only if the total required resources are within these quota limits.
+.. Apply the configuration to create the `cluster-queue` object:
++
+[source,bash]
+----
+$ oc apply -f cluster_queue.yaml
+----
+
+. Create a local queue that points to your cluster queue, as follows:
+.. Create a file called `local_queue.yaml` and populate it with the following content:
++
+.Example local queue
+[source,bash]
+----
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: LocalQueue
+metadata:
+  namespace: test
+  name: local-queue-test
+  annotations:
+	kueue.x-k8s.io/default-queue: 'true' <1>
+spec:
+  clusterQueue: cluster-queue
```

Comment:
```suggestion
apiVersion: kueue.x-k8s.io/v1beta1
kind: LocalQueue
metadata:
  namespace: test
  name: local-queue-test
  annotations:
    kueue.x-k8s.io/default-queue: 'true'
spec:
  clusterQueue: cluster-queue
```

Reply:
Just a question, would it be possible to avoid `<1>` in the code? So customers can copy it without adjustments. If it is important then it can stay, it is just a suggestion.

Reply:
@sutaakar If you use the Copy function in the code example in the published RHOAI user doc, the callouts are not included in the copied code. Try it [here](https://access.redhat.com/documentation/en-us/red_hat_openshift_ai_self-managed/2-latest/html/working_on_data_science_projects/working-with-distributed-workloads_distributed-workloads#running-distributed-data-science-workloads-from-ds-pipeline_distributed-workloads).

However, this functionality is not provided in the ODH docs, so I had already opened RHOAIENG-2386 to remove the callouts. Not sure if this update will be done in time for 2.9 though.

Reply:
ok, so `<1>` can stay

Reply:
Replaced tabs with spaces, and removed callout <1>.